code/game/verbs/suicide.dm

```diff
@@ -9,6 +9,7 @@
 		return
 	if(confirm == "Yes")
 		suiciding = 1
+		log_game("[key_name(src)] (job: [job ? "[job]" : "None"]) commited suicide at [get_area(src)].")

```

Comment:
The round's timestamp (How much time has passed since the round started) might also be of importance. Without it, a Captain suiciding at roundstart looks the same as one doing so after the emergency shuttle has left.